qcodes_contrib_drivers/drivers/Standa/Standa_10MWA168.py

```diff
@@ -0,0 +1,206 @@
+            raise Exception(self._error_codes[code])
+        else:
+            print("libximc: [%s]: Unknown code: %s" % (function_name, code))
+            raise Exception()
+
+    def command_move(self, device_id, position, u_position):
+        self.dll.command_move(device_id, position, u_position)
+
+    def enumerate_devices(self, probe_flags):
+        enumeration = self.dll.enumerate_devices(probe_flags, b"")
+        return enumeration
+
+    def get_device_name(self, device_enumeration, device_index):
+        device_name = self.dll.get_device_name(device_enumeration, device_index)
+        return device_name
+
+    def get_position(self, device_id, get_position):
+        self.dll.get_position(device_id, get_position)
+
+    def get_status(self, device_id, status):
+        self.dll.get_status(device_id, status)
+
+    def open_device(self, device_name):
+        device_id = self.dll.open_device(device_name)
+        return device_id
+
+
+class Standa_10MWA168(Instrument):
+
+    def __init__(self, name, serial_number, dll_path=None, **kwargs):
+        super().__init__(name, **kwargs)
+
+        # link to dll
+        self.libximc = libximc(dll_path=dll_path)
+
+        # instrument constants
+        self.filter_wheel_1 = [-1, 0, 1, -1, 1, 2, -1, 1, 2, 3, 3, 3, 1, 2, 4, 4, 1, 2, 5, 5, 4, 1, 2, 6, 6, 6, 2, 5, 6, 6, 6]
+        self.filter_wheel_2 = [-1, 0, -1, 1, 1, 1, 2, 2, 2, -1, 1, 2, 3, 3, -1, 2, 4, 4, -1, 1, 3, 5, 5, -1, 1, 2, 6, 4, 4, 5, 6]
+        self.offset_wheel_1 = 0.
+        self.offset_wheel_2 = 62.
+        self.revolution = 200.
+        self.distance = self.revolution / 8.
+
+        # initialization
+        self.serial_number = serial_number
+        device_enumeration = self.libximc.enumerate_devices(0)
+        enumeration_name = self.libximc.get_device_name(device_enumeration, 0)
+        self.device_id = self.libximc.open_device(enumeration_name)
+
+        # add parameters
+        self.add_parameter('transmittance',
+                           set_cmd=self.set_transmittance,
+                           label='Transmittance',
+                           val_mapping={
+                               1: 0, 0: 1, 9.0e-1: 2, 8.0e-1: 3, 7.2e-1: 4, 4.0e-1: 5, 3.0e-1: 6, 2.7e-1: 7,
+                               1.5e-1: 8, 1.0e-1: 9, 8.0e-2: 10, 3.0e-2: 11, 2.7e-2: 12, 1.5e-2: 13, 1.0e-2: 14,
+                               3.0e-3: 15, 2.7e-3: 16, 1.5e-3: 17, 1.0e-3: 18, 8.0e-4: 19, 3.0e-4: 20, 2.7e-4: 21,
+                               1.5e-4: 22, 1.0e-4: 23, 8.0e-5: 24, 3.0e-5: 25, 1.5e-5: 26, 3.0e-6: 27, 3.0e-7: 28,
+                               3.0e-8: 29, 3.0e-9: 30})
+
+        self.add_parameter('position',
+                           set_cmd=self.set_position,
+                           get_cmd=self.get_position,
+                           get_parser=float,
+                           label='Position')
+
+        self.add_parameter('status',
+                           get_cmd=self.get_status,
```

Comment:
if parameters are supposed to be used, and not the `set_*` methods explicitly, then please make the `set_` methods private (with underscore in the front).